### ISSUE TITLE
Map tiles are empty outside of France

### ISSUE THREAD

**Issue by wetneb** [bleibtoffen/bleibtoffen#103]
The bleibtoffen.de website seems to be designed for Germany but the map tiles it uses seem to only work in France. For instance, when looking up Berlin on bleibtoffen.de:
https://www.bleibtoffen.de/@52.506902,13.424401,10.29

**Comment by chnuessli** [comment]
Hello @wetneb 
Currently the service at D/AT/CH is closed, only the French instance has relaunched because there is a lockdown.

**Comment by wetneb** [comment]
Ok, perhaps it would be worth making it clearer that `bleibtoffen.de` is closed on the website itself? It is a bit disappointing otherwise :)

**Comment by datendelphin** [comment]
yes that would be a good idea. We should create a simple page explaining that it is down.

**Comment by chnuessli** [comment]
When the frech site was down, there was a translated page that shows in german that the page was down. Can we activate it again @caugner ?

**Comment by datendelphin** [comment]
We could serve it as a static page ourselves, if someone still has a copy of it, or if we can recreate one

**Comment by datendelphin** [comment]
Static site should now be up. Is that ok?

**Comment by wetneb** [comment]
For me, nothing has changed.

**Comment by wetneb** [comment]
(By the way it's not clear to me why the service should not be available in Germany: there are a lot of local restrictions, restaurants can only do take aways, which probably influences their opening times…)

**Comment by chnuessli** [comment]
@wetneb How do you access the website?
@datendelphin Thank you for your work. for me, it's ok like that.

**Comment by wetneb** [comment]
@chnuessli I go to https://www.bleibtoffen.de/ and I see a working webapp there (except for the empty map tiles). I still get the webapp even if I hard refresh.

**Comment by datendelphin** [comment]
Ah I forgot, the .de domain uses a different DNS server. @giggls could you switch the .de domain to point to 95.128.36.189?
availability in Germany: If there were enough volunteers we could reopen the service running this time on our own servers. But it is significant work (set it up on our own servers, previously it was run courtesy of the French on their servers) (also, enter all the complex rules of the current situation) and I didn't see much interest yet.

**Comment by chnuessli** [comment]
IMO the interest was much bigger on the first wave. There was a lockdown across whole countries and even then, the interest wasn't there in CH/AT. In Germany i think the interest was there and there could be some new contributors.

**Comment by wetneb** [comment]
Fair enough, I don't want to be pushy on this!
